src/lightning_lite/utilities/types.py

```diff
@@ -30,14 +30,11 @@
 if torch.distributed.is_available():
     from torch.distributed import ProcessGroup, ReduceOp
 
-    if _TORCH_GREATER_EQUAL_1_13:
-        _TORCH_REDUCE_OP = ReduceOp.RedOpType
-    else:
-        _TORCH_REDUCE_OP = ReduceOp
+    RedOpType = ReduceOp.RedOpType if _TORCH_GREATER_EQUAL_1_13 else object
```

Comment:
```suggestion
    RedOpType = ReduceOp.RedOpType if _TORCH_GREATER_EQUAL_1_13 and hasattr(ReduceOp, "RedOpType") else object
```
it comes from https://github.com/Lightning-AI/lightning/pull/15128#issuecomment-1285054975

Reply:
I don't think this fix is correct. Insetad, we should undo `use_base_version` now that 1.13 is finalized